src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -101,37 +126,66 @@
 
 {# Print the result #}
 
-<div 
+<{{ _group_tag }} 
   class="{{ _css_class }}"
+  {% if _input.input_type == 'radio' or _input.input_type == 'checkbox' or _input.input_type == 'rating-field' %}
+    aria-describedby="{{ _aria_describedby }}"
+    {% if _invalid %}
+      aria-invalid="true"
+    {% endif %}
+  {% endif %}
   {{- _extra_attributes|raw -}}
 >
   {% if _label is not empty %}
     <{{ _label_tag }}
     {% if _id %}
       for="{{ _id }}"
+      id="{{ _id }}-label"
     {% endif %}
       class="{{- _label_class -}}
       {{- _invalid ? ' ecl-form-label--invalid' -}}
       {{- _disabled ? ' ecl-form-label--disabled' -}}"
     >
       {%- block label _label -%}
       {%- if _required and _required_text is not empty -%}
-        <span class="ecl-form-label__required">{{ _required_text }}</span>
+        <span 
+          class="ecl-form-label__required"
+          {% if _label_aria_required %}
+            aria-label="{{ _label_aria_required }}"
+          {% endif %}
+        >{{ _required_text }}</span>
       {%- elseif not _required and _optional_text is not empty -%}
-        <span class="ecl-form-label__optional">{{ _optional_text }}</span>
+        <span 
+          class="ecl-form-label__optional"
+          {% if _label_aria_optional %}
```

Comment:
it seems to me we should add some white space controls here, {%- if _label_aria_optional -%}  {%- if _label_aria_required -%} etc...